content/reference/dos-mitigation.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "DOS Mitigation"
```

Comment:
Can we have a table of contents?  Or does the docs website autogenerate?  I didn't see one in https://bafybeiegvuphbhruoy6ntjhgh5xfmweye7ubeshinyv5lugdhgxml6dice.on.fleek.co/reference/dos-mitigation/

Reply:
added one